src/pint/models/jump.py

```diff
@@ -127,3 +131,10 @@ def print_par(self):
             jump_par = getattr(self, jump)
             result += jump_par.as_parfile_line()
         return result
+
+    def get_number_of_jumps(self):
+        return len(self.jumps)
+
+    def get_jump_param_objects(self):
```

Comment:
@champagne-cmd can you add brief docstrings to these functions?

Reply:
Certainly!